src/app/screens/NewArticle/index.tsx

```diff
@@ -0,0 +1,130 @@
+
+const { MIN_LENGHT_FIELD, MAX_TITLE_LENGHT, MAX_DESCRIPTION_LENGHT, MAX_BODY_LENGHT } = fiedlsValidations();
+
+function NewArticle() {
+  const { handleSubmit, control, setValue, trigger } = useForm<NewArticleValues>({ mode: 'all' });
+  const [items, setItems] = useState<string[]>([]);
+  const handleDeleteTag = (index: number) => {
+    const newItems = items.filter((_, key) => key !== index);
+    setItems(newItems);
+  };
+
+  const renderItem: ListRenderItem<string> = ({ item, index }) => (
+    <Tag text={item} index={index} onDeleteTag={handleDeleteTag} />
+  );
+  // @TODO: do the quest when you already have the token in the headers
+  const handleNewArticle = (values: NewArticleValues) => {
+    trigger();
+    console.log(values);
+  };
+
+  useEffect(() => {
+    setValue(FIELDS.tagList, items);
+  }, [items, setValue]);
+
+  return (
+    <View style={styles.container}>
+      <KeyboardAwareScrollView bounces={false} enableOnAndroid>
+        <CustomText green bold style={styles.title}>
+          Create article
+        </CustomText>
+        <ControlledCustomTextInput
+          testIDProp={testIds.titleInput}
+          control={control}
+          label={i18next.t('NEW_ARTICLE:TITLE')}
+          name={FIELDS.title}
+          placeholder={i18next.t('NEW_ARTICLE:PLACEHODER_TITLE')}
+          labelStyle={styles.labelStyle}
+          rules={{
+            ...validateRequired,
+            ...validateMinLength(MIN_LENGHT_FIELD),
+            ...validateMaxLength(MAX_TITLE_LENGHT),
+            ...validateAlphanumeric
+          }}
+          maxLength={MAX_TITLE_LENGHT}
+        />
+        <ControlledCustomTextInput
+          testIDProp={testIds.descriptionInput}
+          control={control}
+          labelStyle={styles.labelStyle}
+          label={i18next.t('NEW_ARTICLE:DESCRIPTION')}
+          name={FIELDS.description}
+          placeholder={i18next.t('NEW_ARTICLE:PLACEHOLDER_DESCRIPTION')}
+          rules={{
+            ...validateRequired,
+            ...validateMinLength(MIN_LENGHT_FIELD),
+            ...validateMaxLength(MAX_DESCRIPTION_LENGHT),
+            ...validateAlphanumeric
+          }}
+          maxLength={MAX_DESCRIPTION_LENGHT}
+        />
+
+        <ControlledCustomTextInput
+          testIDProp={testIds.bodyInput}
+          control={control}
+          multiline
+          label={i18next.t('NEW_ARTICLE:BODY')}
+          labelStyle={styles.labelStyle}
+          name={FIELDS.body}
+          placeholder={i18next.t('NEW_ARTICLE:PLACEHOLDER_BODY')}
+          rules={{
+            ...validateRequired,
+            ...validateMinLength(MIN_LENGHT_FIELD),
+            ...validateMaxLength(MAX_BODY_LENGHT)
+          }}
+          maxLength={MAX_BODY_LENGHT}
+        />
+
+        <ControlledCustomTextInput
+          testIDProp={testIds.tagInput}
+          control={control}
+          labelStyle={styles.labelStyle}
+          label={i18next.t('NEW_ARTICLE:TAGS')}
+          name={'addTag'}
```

Comment:
field missed

Reply:
This field is not needed in the values, it is only to add the label